.github/workflows/release.yml

```diff
@@ -178,17 +178,16 @@ jobs:
 
   images:
     needs: published
-    uses: infinispan/infinispan-images/.github/workflows/release.yml@main
+    uses: ./.github/workflows/release_image.yml
     secrets: inherit
     with:
-      branch: main
-      repository: infinispan/infinispan-images
-      ispnVersion: ${{ inputs.version }}
+      branch: ${{ inputs.branch }}
+      version: ${{ inputs.version }}
       type: ${{ inputs.type }}
 
   operator-main:
     needs: images
-    if: contains(inputs.version, '.Final')
+    if: contains(inputs.version, '.Dev')
```

Comment:
This seems wrong, previously we only called this job if it was a .Final release and now only .Dev? Maybe we should use the type here, i.e if `!prerelease` call the workflow.